sample-code/spring-app/pom.xml

```diff
@@ -157,6 +157,10 @@
       <artifactId>assertj-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.sap.cloud.sdk.cloudplatform</groupId>
+      <artifactId>cloudplatform-connectivity</artifactId>
+    </dependency>
```

Comment:
(Major)
Wait, why is this necessary?

Reply:
Without the declaration, the maven dependency plugin with goal "analyze-only" reports "Used undeclared dependencies found"